### ISSUE TITLE
Gradle. StackTraceAnalyser - Couldn't find class during Stack analysis

### ISSUE THREAD

**Issue by QAutomatron** [serenity-bdd/serenity-core#804]
I try to build Serenity project with Gradle. Everything works fine, except lot of debug messages from StackTraceAnalyser.

Serenity version = 1.4.0
Gradle version = 3.5

My `build.gradle` :
```
apply plugin: 'java'
apply plugin: 'idea'
apply plugin: 'maven'
apply plugin: 'net.serenity-bdd.aggregator'

group = 'com.project'
version = '1.0'

ext {
    serenity_version = "1.4.0"
}

buildscript {
    repositories {
        mavenLocal()
        jcenter()
        mavenCentral()
    }
    dependencies {
        classpath("net.serenity-bdd:serenity-gradle-plugin:1.4.0")
    }
}

dependencies {
    compile group: 'net.serenity-bdd', name: 'serenity-core', version:serenity_version
    compile group: 'net.serenity-bdd', name: 'serenity-junit', version:serenity_version
    compile group: 'junit', name: 'junit', version:'4.12'
    compile group: 'org.slf4j', name: 'slf4j-simple', version:'1.7.21'
    compile group: 'io.appium', name: 'java-client', version:'5.0.0-BETA8'
    compile group: 'io.codearte.jfairy', name: 'jfairy', version:'0.5.6'
    compile group: 'com.google.inject.extensions', name: 'guice-assistedinject', version:'3.0'
    compile group: 'org.iban4j', name: 'iban4j', version:'2.1.1'
    compile group: 'io.rest-assured', name: 'rest-assured', version:'3.0.2'
}

gradle.startParameter.continueOnFailure = true
sourceCompatibility = 1.8
targetCompatibility = 1.8

tasks.withType(JavaCompile) {
    options.encoding = 'UTF-8'
}

repositories {
    mavenLocal()
    mavenCentral()
    jcenter()
}

clean {
    delete 'target'
}

test {
    jvmArgs '-Dfile.encoding=UTF-8 -Xms128m -Xmx512m'
    maxParallelForks = 1
    scanForTestClasses true
    testLogging {
        showStandardStreams = true
        exceptionFormat "full"
        events "started", "passed", "skipped", "failed", "standardOut", "standardError"
        displayGranularity = 0
    }
    forkEvery = 0

    filter {
        includeTestsMatching "*_Test"
    }

    // properties
    systemProperties System.getProperties()
    systemProperty 'webdriver.timeouts.implicitlywait', 10000
    systemProperty 'serenity.timeout', 10
}
```

 Log:
```
    12:51:18 DEBUG StepInterceptor - STARTING STEP: LoginHelper.login - skipContactsPermission
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ContextClassLoaderDispatch
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ContextClassLoaderDispatch
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.worker.TestWorker
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.api.internal.tasks.testing.worker.TestWorker
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
    12:51:18 DEBUG StackTraceAnalyser - Couldn't find class during Stack analysis: org.gradle.internal.dispatch.ReflectionDispatch
12:51:18 DEBUG StepInterceptor - STARTING STEP: PermissionScreenSteps.skipContactsPermission - screenShouldBeOpened
```

**Comment by wakaleo** [comment]
Fixed in 1.4.1-rc.4

**Comment by JohanAval123** [comment]
I have 2.0.70 version and have the same mistake
